packages/labs/cli/src/lib/init/element-starter/templates/tsconfig.json.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {FileTree} from '@lit-labs/gen-utils/lib/file-utils.js';
+
+export const generateTsconfig = (): FileTree => {
+  return {
+    'tsconfig.json': `{
+  "compilerOptions": {
+    "target": "es2022",
+    "module": "es2022",
+    "lib": ["es2022", "DOM", "DOM.Iterable"],
+    "declaration": true,
+    "declarationMap": true,
+    "sourceMap": true,
+    "inlineSources": true,
+    "rootDir": "src",
+    "outDir": "lib",
```

Comment:
Let's consider the import paths we want... do we want the `/lib/` in there if someone's importing elements directly? How does this grow if the package author adds more elements?

Reply:
having outdir as "." makes this quite annoying to scale as you'd have to know to manually update the gitignore as you add components. Granted you have to do this already for package exports, but it would be worse if you had to chase this around in more than once place.

Additionally this template is for a single component – maybe we should handle this in the design system?

Reply:
Happy to follow this up in a later PR if deemed necessary – sorta just want to get this PR in for now, and it's not like it's  user-facing quite yet